src/pages/scheduler/containers/SchedulerContainer.tsx

```diff
@@ -22,10 +23,11 @@ export function SchedulerContainer(): JSX.Element {
     () => denormalizeCourseEvents(coursesResult.status === 'loaded' ? coursesResult.data : []),
     [coursesResult]
   );
+  const smallScreen = useSmallScreen();
 
   return (
     <Flex flexGrow={1} height="100%" overflow="hidden" direction={'column'}>
-      <Box w="100%" height="100%" px="3" py="2">
+      <Box w="100%" height="100%" px={smallScreen ? '0' : '3'} py={smallScreen ? '1' : '2'}>
```

Comment:
Can we maybe try px=1, pb=0, and py stays at 2 for small screens? I think it's a bit tight with no padding on the sides but just a lil would be good, and then it would be nice if it went all the way to the bottom.